apps/sms/test/marionette/mocks/mock_split_view_mode.js

```diff
@@ -0,0 +1,14 @@
+/* global Services */
+'use strict';
+
+Services.obs.addObserver(function(document) {
+  if (!document || !document.location ||
+      document.location.origin !== 'app://sms.gaiamobile.org') {
+    return;
+  }
+
+  // Switch to split-view mode Inbox instead.
+  if(document.location.pathname === '/index.html') {
```

Comment:
> Maybe ask Johan ? 

Yeah, will ask him, thanks! :)

> just replace the manifest on the fly before generating the profile and launching mulet...

I believe it will be a pain since we want to run tests for both versions on the same profile....

> if we could add hidden entry points to our manifest,

Hmm interesting, depending on how it will look like. But generally I'd rather make hacky test than hack real manifest :p